src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas3/Cas3SoftDeleteApplicationTest.kt

```diff
@@ -0,0 +1,100 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.integration.cas3
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.integration.givens.givenAUser
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.integration.givens.givenAnOffender
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.UserEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.UserRole
+import java.time.LocalDate
+import java.time.OffsetDateTime
+import java.util.UUID
+
+class Cas3SoftDeleteApplicationTest : IntegrationTestBase() {
```

Comment:
Tests will run a little faster if the test class extends `InitialiseDatabasePerClassTestBase` and you setup an application in a `@BeforeAll` function